estraverse-fb missing

I just downloaded, installed and attempted to run 2.0.0-alpha.1.  "npm start" complained that estraverse-fb was missing.  I did a save-dev install of estraverse-fb, and now all appears to be fine.  I see no errors in the "npm install" log, and no references to estraverse-fb.

err, same issues here. 

Cannot find module 'estraverse-fb'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)

It's not an issue with this starter kit but a related dependency. I'm away on vacation and unable to submit a patch, so I suggest installing locally as a temporary solution or waiting for the appropriate babel/eslint fix since that's the cause, unless you'd like to submit a PR.

Sent from my iPhone

> On Mar 6, 2016, at 4:20 PM, Tony Tai Nguyen <notifications@github.com> wrote:
> 
> err, same issues here.
> 
> Cannot find module 'estraverse-fb'
> at Function.Module._resolveFilename (module.js:339:15)
> at Function.Module._load (module.js:290:25)
> at Module.require (module.js:367:17)
> 
> —
> Reply to this email directly or view it on GitHub.
>

--- a/package.json
+++ b/package.json
@@ -102,7 +102,7 @@
     "css-loader": "^0.23.0",
     "cssnano": "^3.3.2",
     "enzyme": "^2.0.0",
-    "eslint": "^2.1.0",
+    "eslint": "~2.2.0",
     "eslint-config-standard": "^5.1.0",
     "eslint-config-standard-react": "^2.2.0",
     "eslint-loader": "^1.1.1",

